code/modules/antagonists/highlander/highlander.dm

```diff
@@ -10,13 +10,17 @@
 	ADD_TRAIT(L, TRAIT_NOGUNS, "highlander")
 	ADD_TRAIT(L, TRAIT_NODISMEMBER, "highlander")
 	ADD_TRAIT(L, TRAIT_SHOCKIMMUNE, "highlander")
+	ADD_TRAIT(L, TRAIT_NOFIRE, "highlander")
+	ADD_TRAIT(L, TRAIT_NOBREATH	, "highlander")
 	REMOVE_TRAIT(L, TRAIT_PACIFISM, ROUNDSTART_TRAIT)
 
 /datum/antagonist/highlander/remove_innate_effects(mob/living/mob_override)
 	var/mob/living/L = owner.current || mob_override
 	REMOVE_TRAIT(L, TRAIT_NOGUNS, "highlander")
 	REMOVE_TRAIT(L, TRAIT_NODISMEMBER, "highlander")
 	REMOVE_TRAIT(L, TRAIT_SHOCKIMMUNE, "highlander")
+	REMOVE_TRAIT(L, TRAIT_NOFIRE, "highlander")
+	REMOVE_TRAIT(L, TRAIT_NOBREATH	, "highlander")
```

Comment:
Add a `HIGHLANDER_TRAIT` define, and then change the third argument to this macro to `HIGHLANDER_TRAIT`.